### PR TITLE
Gif block: Update placeholder

### PR DESCRIPTION
**Pull request by scruffian**
#### Changes proposed in this Pull Request:

* Makes some changes the the placeholder for the Gif block to make it more aligned with other embed blocks

#### Testing instructions:

* Create a post
* Add a Gif block
* Check that the placeholder looks like this:
Screenshot 2020-02-04 at 11 49 16
* Check that the block preview looks like this:
Screenshot 2020-02-04 at 11 49 25

#### Proposed changelog entry for your changes:

* no changelog

### CODE REVIEW

**Comment by apeatling** [comment]
Placeholder looked like this for me, using Gutenberg master. All looked aligned however:

Screen Shot 2020-02-07 at 10 55 36 AM

**Comment by apeatling** [comment]
Preview looked like this, matching your image above:

Screen Shot 2020-02-07 at 10 56 40 AM

File: extensions/blocks/gif/edit.js [hunk 1]
```diff
@@ -136,11 +136,11 @@ class GifEdit extends Component {
 				onSubmit={ this.onFormSubmit }
 				ref={ this.textControlRef }
 			>
-				<TextControl
-					className="wp-block-jetpack-gif_input"
-					label={ INPUT_PROMPT }
-					placeholder={ INPUT_PROMPT }
-					onChange={ value => setAttributes( { searchText: value } ) }
+				<input
```
Comment (apeatling) [inline extensions/blocks/gif/edit.js]: Was there an issue with the TextControl component? Is there anything there that can be fixed in core Gutenberg?

Comment (scruffian) [reply extensions/blocks/gif/edit.js]: I'm not sure. The embed block in core doesn't use it, and we are just following that example. Probably worth asking more about that though...

Comment (scruffian) [reply extensions/blocks/gif/edit.js]: I asked here but didn't get any insight! https://wordpress.slack.com/archives/C02QB2JS7/p1581590282345300

**Review by jeherve — APPROVED** [review]
This looks good to me. 🚢

**Comment by scruffian** [comment]
r202851-wpcom

**Pull request merged**